docs/_sass/_main.scss

```diff
@@ -182,6 +183,190 @@ button {
     align-content: center;
 }
 
+.logo {
+    width: 80%;
+}
+
+.search-icon {
+    margin: auto 0px;
+}
+
+.flex {
+    display: -webkit-box;
+    display: -moz-box;
+    display: -ms-flexbox;
+    display: -moz-flex;
+    display: -webkit-flex;
+    display: flex;
+    -webkit-flex-flow: row wrap;
+    flex-flow: row wrap;
+    align-content: space-between;
+}
+
+#sidebar-search {
+    background-color: $color-green-appBG;
+    width: 400px;
+    height: 100vh;
+    position: fixed;
+    display: block;
+    top: 0;
+    right: 0;
+}
+
+@media only screen and (max-width: $breakpoint-tablet) {
+    #sidebar-search {
+        width: 100%;
+    }
+}
+
+.searchbar-title-wrapper {
+    padding: 20px;
+}
+
+.search-title {
+    font-size: 120%;
+}
+
+#toggle-search-close {
+    margin: auto;
+    margin-left: auto;
+    margin-right: 0px;
+}
+
+#sidebar-layer {
+    position: fixed;
+    /* Sit on top of the page content */
+    display: none;
+    /* Hidden by default */
+    width: 100%;
+    /* Full width (cover the whole page) */
+    height: 100%;
+    /* Full height (cover the whole page) */
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: #1b574470;
+    z-index: 2;
+    /* Specify a stack order in case you're using a different order for other elements */
+}
+
+#___gcse_0 {
+    margin-left: 20px;
+}
+
+input#gsc-i-id1.gsc-input {
+    background-color: $color-green-appBG;
+    color: #E7ECE9;
+    font-family: "ExpensifyNeue", "Segoe UI Emoji", "Noto Color Emoji" !important;
+}
+
+#gsc-iw-id1 {
```

Comment:
What is this element?

Reply:
Yupp comment added

Reply:
The comment isn't very specific though. For example, what is this particular ID for? A search box? Something else?